contracts/vaults/TreasuryVault/RibbonTreasuryVault.sol

```diff
@@ -369,9 +313,20 @@ contract RibbonTreasuryVault is
      */
     function setCap(uint256 newCap) external onlyOwner {
         require(newCap > 0, "!newCap");
+
+        uint256 previousCap = vaultParams.cap;
+        vaultParams.cap = _getCap104(newCap);
```

Comment:
I don't really see a good reason refactoring this out into a separate function imo. Makes it hard to read